draft-ietf-ohai-ohttp.md

```diff
@@ -482,8 +482,11 @@ HPKE Symmetric Algorithms:
 
 The "application/ohttp-keys" format is a media type that identifies a serialized
 collection of key configurations. The content of this media type comprises one
-or more key configuration encodings (see {{key-config}}) that are concatenated;
-see {{iana-keys}} for a definition of the media type.
+or more key configuration encodings (see {{key-config}}).  Each encoded
+configuration is prefixed with a two byte integer in network byte order that
+indicates the length of the key configuration in bytes.  The length-prefixed
+encodings are concatenated.  See {{iana-keys}} for a definition of the media
```

Comment:
```suggestion
encodings are concatenated to form a list.  See {{iana-keys}} for a definition of the media
```

Reply:
Do we want to say anything about what receivers should do if, say, one of the configs in the list is malformed but others are not?

Reply:
Since the entire point of this change seems to be to allow ignoring unrecognized configs, it seems there should at least be some language about doing that, if not ignoring recognized but malformed configs.

Reply:
My implementation treats the entire thing as invalid if there are any decode errors (aside from those that follow the public key for an unknown KEM, that is, because those errors are essentially impossible to locate).

Reply:
Similarly, my implementation in Google QUICHE currently rejects all of them unless they're all parsed as vaild, so I agree that that's a good way to go.